app/src/main/java/org/simple/clinic/editpatient/EditPatientConfig.kt

```diff
@@ -1,3 +1,14 @@
 package org.simple.clinic.editpatient
 
-data class EditPatientConfig(val deletePatientFeatureEnabled: Boolean = false)
+import org.simple.clinic.remoteconfig.ConfigReader
+import javax.inject.Inject
+
+data class EditPatientConfig(
+    val deletePatientFeatureEnabled: Boolean
+) {
+
+  @Inject
+  constructor(configReader: ConfigReader) : this(
+      deletePatientFeatureEnabled = configReader.boolean("delete_patient_feature_enabled", false)
```

Comment:
Why is this disabled by default? Since the feature is complete, we can enable it by default, right?

Reply:
Ah yes, I forgot to update it after changing it back to the remote flag 😅